packages/cli/src/commands/teams/invite.js

```diff
@@ -163,57 +146,52 @@ export default async function ({
       }
     }
     let elapsed;
-    let stopSpinner;
     if (email) {
       elapsed = stamp();
-      stopSpinner = wait(inviteUserPrefix + email);
+      output.spinner(inviteUserPrefix + email);
       try {
         // eslint-disable-next-line no-await-in-loop
         const { name, username } = await teams.inviteUser({
           teamId: currentTeam.id,
           email,
         });
-        stopSpinner();
         const userInfo = name || username;
         email = `${email}${userInfo ? ` (${userInfo})` : ''} ${elapsed()}`;
         emails.push(email);
-        console.log(`${chalk.cyan(chars.tick)} ${sentEmailPrefix}${email}`);
+        output.log(`${chalk.cyan(chars.tick)} ${sentEmailPrefix}${email}`);
         if (hasError) {
           hasError = false;
-          process.stdout.write(eraseLines(emails.length + 2));
-          console.log(
-            info(
-              introMsg ||
-                `Inviting team members to ${chalk.bold(currentTeam.name)}`
-            )
+          process.stderr.write(eraseLines(emails.length + 2));
```

Comment:
Do you need a `output.stopSpinner()` here before erasing lines?

Reply:
Not in this case because `output.log()` on line 161 stops the spinner.